glide-core/src/client/value_conversion.rs

```diff
@@ -121,6 +122,46 @@ pub(crate) fn convert_to_expected_type(
         ExpectedReturnType::BulkString => Ok(Value::BulkString(
             from_owned_redis_value::<String>(value)?.into(),
         )),
+        ExpectedReturnType::JsonToggleReturnType => match value {
+            Value::Array(array) => {
+                let new_array: Result<Vec<_>, _> = array
+                    .into_iter()
+                    .map(|arr| match arr {
+                        Value::Nil => Ok(Value::Nil),
+                        _ => match from_owned_redis_value::<bool>(arr.clone()) {
+                            Ok(boolean_value) => Ok(Value::Boolean(boolean_value)),
+                            _ => Err((
+                                ErrorKind::TypeError,
+                                "Could not convert value to boolean",
+                                format!("(value was {:?})", arr),
+                            )
+                                .into()),
+                        },
+                    })
+                    .collect();
+
+                match new_array {
+                    Ok(values) => Ok(Value::Array(values)),
+                    Err(err) => Err(err),
+                }
+            }
+            Value::BulkString(s) => match std::str::from_utf8(&s) {
```

Comment:
Does it support booleans only?
If yes - please add a comment; if no - add a TODO

Reply:
Do you mean if it supports String and array only? I want the  command to return True and False instead of 0 and 1 (in case of a JSONPath) and instead of "true" and "false" (legacy path).

https://redis.io/docs/data-types/json/path/

an example:
```
127.0.0.1:6379> json.set doc $ '{"bool": true , "nested": {"bool": false , "nested": {"bool": 10}}}'
OK
127.0.0.1:6379> json.toggle doc $..bool
1) (integer) 0
2) (integer) 1
3) (nil)
127.0.0.1:6379> json.toggle doc bool
"true"
```

Reply:
I see. It is not clear why this function ignores other types - it is worth adding a comment.